_layouts/default.html

```diff
@@ -44,19 +44,26 @@ <h2>Contents:</h2>
 
         <h2>Pipelines:</h2>
         {% for repository in site.github.public_repositories %}
-          {% assign notShown =  "test|ansible-cromwell-database|biowdl.github.io|tasks|pipeline-template" | split: "|" %}
-          {% unless notShown contains repository.name %}
+          <!--{% assign notShown =  "test|ansible-cromwell-database|biowdl.github.io|tasks|pipeline-template" | split: "|" %}-->
```

Comment:
Since splitting based on labels is not possible we should make static list what we do want to show where.
Maybe something like this:
```
<!--{% assign multisample =  "germlins-DNA|RNA-seq|gams" | split: "|" %}--> 
<!--{% assign multisampleExperimental =  "something_else" | split: "|" %}--> 
<!--{% assign single =  "QC" | split: "|" %}--> 
<!--{% assign singleExperimental =  "BamMetrics" | split: "|" %}--> 
```

This way if we make a new repository this page will automaticly added here, even it's just a testing repo